frappe/public/js/frappe/ui/page.js

```diff
@@ -375,15 +375,20 @@ frappe.ui.Page = Class.extend({
 			$(this.inner_toolbar).removeClass("hide");
 
 			if (!this.is_in_group_button_dropdown($group.find(".dropdown-menu"), 'li', label)) {
-				return $('<li><a>'+label+'</a></li>')
+				return $('<li><a data-label="'+label+'">'+label+'</a></li>')
```

Comment:
we also need to escape the label, incase someone wants to use quotes in the label!

maybe use:

`let label_id = encodeURIComponent(label)` ?